omod/src/main/webapp/pages/metadata/roles/templates/edit.gsp

```diff
@@ -0,0 +1,81 @@
+<h2 ng-hide="role.uuid">${ ui.message("adminui.role.addNewRole.title") }</h2>
+<h2 ng-show="role.uuid">${ ui.message("adminui.role.editRole.title") }</h2>
+
+<form class="simple-form-ui" name="roleForm" novalidate ng-submit="save()">
+    <p>
+        <label>${ ui.message("adminui.role.role") }</label>
+        <input ng-disabled="role.uuid" ng-model="role.name" ng-class="{ adminuiDisabled: role.uuid }" required/>
+    </p>
+    <p>
+        <label>${ ui.message("adminui.role.description") }</label>
+        <input ng-model="role.description" required/>
+    </p>
+    <div ng-if="dependantRoles.length > 0">
+	    <p>
+			${ ui.message("adminui.role.containedRoles.label") } {{role.name}}
+	    </p>
+	    <p>
+            <table>
+                <tbody>
+                    <tr ng-repeat="(idx, val) in dependantRoles" ng-hide="idx % 2">
+                        <td >
+                            {{dependantRoles[idx].name}}
+                        </td>
+                        <td ng-hide="!dependantRoles[idx + 1]">
+                            {{dependantRoles[idx + 1].name}}
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+	    </p>
+    </div>
+    <p>
+        <label>${ ui.message("adminui.role.inheritedRoles") }</label>
+    </p>
+    <p>
+        <table>
+            <tbody>
+                <tr ng-repeat="(idx, val) in roles" ng-hide="idx % 2">
+                    <td >
+                        <input type="checkbox" ng-model="inheritedRoles[idx]"
+        					ng-click="selectInheritedRole()">
+                        {{roles[idx].name}}
+                    </td>
+                    <td ng-hide="!roles[idx + 1]" >
+                        <input type="checkbox" ng-model="inheritedRoles[idx + 1]"
+        					ng-click="selectInheritedRole()">
+                        {{roles[idx + 1].name}}
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+    </p>
+    <p>
+        <label>${ ui.message("adminui.role.privileges") }</label>
+    </p>
+    <p>
+        <label>${ ui.message("adminui.role.inheritedPrivileges.label") }</label>
```

Comment:
This is the label that says "Privileges inherited from other roles are disabled", right? This is not correct when creating a new role, so you should only